project/src/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from 'react-router-dom';
+import styles from './not-found.module.css';
+import Header from '../../components/header/header';
+
+function NotFound(): JSX.Element {
+  const navigate = useNavigate();
+
+  return (
+    <div className="page page--gray">
+      <Header isLogged={false} />
+
+      <main className={`${styles.page__error} error`}>
+        <section>
+          <img className={`${styles.error__img}`} src="img/oops.svg" alt="Sad emoji" width={200} height={200} />
```

Comment:
Либо если бэм и ни шагу назад то 
```suggestion
          <img className={styles['error__img']} src="img/oops.svg" alt="Sad emoji" width={200} height={200} />
```